references.bib

```diff
@@ -0,0 +1,66 @@
+@inproceedings{petrillo2010agility,
+  title={Is agility out there?: agile practices in game development},
+  author={Petrillo, Fabio and Pimenta, Marcelo},
+  booktitle={Proceedings of the 28th ACM International Conference on Design of Communication},
+  pages={9--15},
+  year={2010},
+  organization={ACM}
+}
+@inproceedings{koivisto2007using,
+  title={Using prototypes in early pervasive game development},
+  author={Koivisto, Elina MI and Suomela, Riku},
+  booktitle={Proceedings of the 2007 ACM SIGGRAPH symposium on Video games},
+  pages={149--156},
+  year={2007},
+  organization={ACM}
+}
+@inproceedings{al2014towards,
+  title={Towards agent-based agile approach for game development methodology},
+  author={Al-Azawi, Rula and Ayesh, Aladdin and Obaidy, Mohaned Al},
+  booktitle={Computer Applications and Information Systems (WCCAIS), 2014 World Congress on},
+  pages={1--6},
+  year={2014},
+  organization={IEEE}
+}
+@inproceedings{popli2014prioritising,
+  title={Prioritising user stories in agile environment},
+  author={Popli, Rashmi and Chauhan, Naresh and Sharma, Hemant},
+  booktitle={Issues and Challenges in Intelligent Computing Techniques (ICICT), 2014 International Conference on},
+  pages={515--519},
+  year={2014},
+  organization={IEEE}
+}
+@inproceedings{chauhan2015reduce,
+  title={How to reduce user story reopen count in Scrum development?},
+  author={Chauhan, Vinod Kumar},
+  booktitle={Computing for Sustainable Global Development (INDIACom), 2015 2nd International Conference on},
+  pages={2204--2208},
+  year={2015},
+  organization={IEEE}
+}
+@inproceedings{lucassen2015forging,
+  title={Forging high-quality user stories: towards a discipline for agile requirements},
+  author={Lucassen, Garm and Dalpiaz, Fabiano and van der Werf, Jan Martijn EM and Brinkkemper, Sjaak},
+  booktitle={2015 IEEE 23rd international requirements engineering conference (RE)},
+  pages={126--135},
+  year={2015},
+  organization={IEEE}
+}
+@misc{agilehard,
+    title={Agile Game Development is Hard},
+    author={{Rob Galanakis}},
+    howpublished={[Online]. Available: http://www.gamasutra.com/blogs/RobGalanakis/20140219/211185/Agile_Game_Development_is_Hard.php},
+    note={[Accessed: 21-Nov-2016]}
+}
+@misc{agilemanifesto,
+    title={Principles behind the Agile Manifesto},
+    author={{Kent Beck, Mike Beedle, Arie van Bennekum, Alistair Cockburn, Ward Cunningham, Martin Fowler, James Grenning, Jim Highsmith, Andrew Hunt, Ron Jeffries, Jon Kern, Brian Marick, Robert C. Martin, Steve Mellor,  Ken Schwaber, Jeff Sutherland, Dave Thomas}},
+    howpublished={[Online]. Available: http://agilemanifesto.org/},
+    note={[Accessed: 21-Nov-2016]}
+}
+@misc{featurecreep,
+    title={Killing Feature Creep Without Ever Saying No},
+    author={{Scott Crabtree}},
+    howpublished={[Online]. Available: http://tinyurl.com/jzbzoev},
+    note={[Accessed: 24-Nov-2016]}
+}
```

Comment:
You've found a lot of conference papers. Are there any journal articles on this topic?